src/components/Tag.tsx

```diff
@@ -12,8 +12,16 @@ interface Props {
   remove: (i: number) => void;
   validator?: (val: string) => boolean;
   removeOnBackspace?: boolean;
+  delimiters?: number[];
 }
 
+export const CloseIcon = () => (
+  <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Add a title for accessibility and tests, like `title="times-icon"` ...

Reply:
![image](https://user-images.githubusercontent.com/23286236/96600597-037aee00-12e9-11eb-88d4-09f8e3b5764d.png)
help?